src/Ninja/G1CP/Content/Tests/test228.d

```diff
@@ -0,0 +1,44 @@
+/*
+ * #228 Gorn can't be asked about after talking to Lares
+ *
+ * There does not seem an easy way to test this fix programmatically, so this test relies on manual confirmation.
```

Comment:
The description of the test is incorrect. Maybe a leftover from a previous manual version of the test.